test/PositionSpec.js

```diff
@@ -20,7 +20,7 @@ describe('Position', function () {
     assert.equal(ReactDOM.findDOMNode(instance).nodeName, 'SPAN');
   });
 
-  it('Should warn about several children', function () {
+  xit('Should warn about several children', function () {
```

Comment:
Shouldn't that be `it` again?

Reply:
ping?

Reply:
this is failing because of the other issues trying to be fixed in #207

Reply:
add a FIXME